src/api/cvc4cpp.cpp

```diff
@@ -27,6 +27,7 @@
 #include "expr/type.h"
 #include "options/main_options.h"
 #include "options/options.h"
+#include "options/smt_options.h"
```

Comment:
I may be mistaken but could it be that this include is not actually needed because you are using `SmtEngine::getOption()` to check whether incremental mode is enabled or not?

Reply:
indeed